lib/apruve/resources/order.rb

```diff
@@ -54,7 +54,6 @@ def initialize(params)
     def validate
       errors = []
       errors << 'merchant_id must be set' if merchant_id.nil?
-      errors << 'payment_term must be supplied' if payment_term.nil?
```

Comment:
I added this validation because we need the corporate account ID. The corporate account ID is passed in the payment_term segment. I'm not sure how you would create an order without this information.